pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Glossary
+---
+
+- **The Graph**: A decentralized protocol for indexing and querying data.
+
+- **Query**: A request for data. In the case of The Graph, a query is a request for data from a subgraph that will be answered by an Indexer.
+
+- **GraphQL**: A query language for APIs and a runtime for fulfilling those queries with your existing data. The Graph uses GraphQL to query subgraphs.
+
+- **Subgraph**: A custom API built on blockchain data that can be queried using the [GraphQL](https://graphql.org/). Developers can build, deploy and publish subgraphs to The Graph's decentralized network. Then, Indexers can begin indexing subgraphs to make them available to be queried by subgraph consumers.
+
+- **Hosted Service**: A temporary scaffold service for building and querying subgraphs while The Graph's decentralized network matures the cost of service, quality of service, and developer experience.
+
+- **Indexers**: Network participants that run indexing nodes to serve queries to subgraphs.
+
+- **Indexer revenue streams**: Indexer's are rewarded in GRT with two components: indexing rewards and query fee rebates.
+
+  1. **Query fee rebates**: Payments from subgraph consumers for serving queries on the network.
+
+  2. **Indexing rewards**: The rewards that Indexers receive for indexing subgraphs. Indexing rewards are generated via new issuance of 3% GRT annually.
```

Comment:
These are listed in the reverse order as mentioned in the previous sentence ("indexing rewards and query fee rebates").